app/src/main/java/org/xbmc/kore/host/HostManager.java

```diff
@@ -132,7 +132,6 @@ public static HostManager getInstance(Context context) {
                 while (cursor.moveToNext()) {
                     int idx = 0;
                     int id = cursor.getInt(idx++);
-                    long updated = cursor.getLong(idx++);
```

Comment:
This should not be removed or you must make sure idx is incremented correctly to get the next element.